doc/source/building/index.rst

```diff
@@ -376,6 +376,37 @@ interface is self-documenting, so please see ``python dev.py --help`` and
 ``python dev.py <subcommand> --help`` for detailed guidance.
 
 
+.. admonition:: IDE support & editable installs
+
+    While the ``dev.py`` interface is our recommended way of working on SciPy,
+    it has one limitation: because of the custom install location, setting
+    breakpoints inside an IDE or debugger does not work as expected. This will
+    only work with an *in-place build* (or "editable install").
+
+    Editable installs are supported. It is important to understand that **you
+    must use either an editable install or dev.py, but not both**. If you use
+    editable installs, you can use ``pytest`` and other development tools
+    directly instead of using ``dev.py``.
+
+    To use an editable install, ensure you start from a clean repository (run
+    ``git clean -xdf`` if you've built with ``dev.py`` before) and have all dependencies set up correctly as described higher up on this page. Then do::
+
+        # Note: the --no-build-isolation is important! meson-python will
+        # auto-rebuild each time Scipy is imported by the Python interpreter.
+        pip install -e . --no-build-isolation
+
+        # To run the tests for, e.g., the `scipy.linalg` module:
+        pytest scipy/linalg
+
+    When making changes to SciPy code, including to compiled code, there is no
+    need to manually rebuild. When you run ``git clean -xdf``, which removes
```

Comment:
```suggestion
    need to manually rebuild or reinstall. When you run ``git clean -xdf``, which removes
```
Since `pip install` looks like an install command.